<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KNewPasswordWidget</class>
 <widget class="QWidget" name="KNewPasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>130</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string comment="@label:textbox">Password:</string>
       </property>
       <property name="buddy">
        <cstring>linePassword</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelVerifyPassword">
       <property name="text">
        <string comment="@label:textbox">&amp;Verify:</string>
       </property>
       <property name="buddy">
        <cstring>lineVerifyPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KPasswordLineEdit" name="linePassword">
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineVerifyPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelStrengthMeter">
       <property name="text">
        <string comment="@label">Password strength &amp;meter:</string>
       </property>
       <property name="buddy">
        <cstring>strengthBar</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="strengthBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPasswordLineEdit</class>
   <extends>QWidget</extends>
   <header>kpasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
